edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -273,15 +272,22 @@ def _is_jwt_cookie_and_session_user_mismatch(self, request):
         - Also adds monitoring details for mismatches.
         - Should only be called for JWT cookies.
         """
-        # adds early monitoring for the JWT LMS user_id
-        jwt_lms_user_id = self._get_and_monitor_jwt_cookie_lms_user_id(request)
+        jwt_username, jwt_lms_user_id = self._get_unsafe_jwt_cookie_username_and_lms_user_id(request)
+
+        # add early monitoring for the JWT LMS user_id for observability for a variety of user cases
+
+        # .. custom_attribute_name: jwt_cookie_lms_user_id
+        # .. custom_attribute_description: The LMS user_id pulled from the
+        #     JWT cookie, or None if the JWT was corrupt and it wasn't found.
+        #     Note that the decoding is unsafe, so this isn't just for valid cookies.
+        set_custom_attribute('jwt_cookie_lms_user_id', jwt_lms_user_id)
```

Comment:
**Note to reviewer**: This attribute already existed, but was moved out of the now renamed `_get_and_monitor_jwt_cookie_lms_user_id`, and its description was updated.

Please mark resolved once read.